### ISSUE TITLE
Setup a new screener screen

### ISSUE THREAD

**Issue by andrewmaier** [jeremiak/call-me-maybe#4]
In order to add a new screener on my site, as a non-technologically inclined design researcher, I'd like a setup a new screener screen.

Since we're using Jekyll, this would essentially be a "new post" screen as far as I can tell. We'd likely want to have some options for adding roles and/or fields. Maybe I can whip up an alternate theme (so that it shows along the top as opposed to a modal popup). Maybe users can control the display % (I think it defaults to 2%.

Drawing on @noahkunin's announcement the other week, I think we might also want to prompt researchers to link to a GitHub post/gist (that details that this study is open to the public should someone want to opt in).

Maybe in the future we could link this to @konklone's Pulse, to show which sites are actively running user tests.

**Comment by andrewmaier** [comment]
I can wireframe this.

**Comment by jeremiak** [comment]
You're totally correct that since we are using Jekyll to build the "embed" code that people would copy/paste into their site code we just need to make a new "Post" and we'll get a new code. I think this is most useful to send as a link to a partner agency so that they can get background and the code in one place.

The roles and fields are all key value pairs between the '---'s on the post page. This is called front-matter in Jekyll land. I'm not sure we need a whole view for this though, it depends on who will be creating these codes. I think for now we can just create them using Github's editor or prose.io since it will 18Fers who will be creating the codes. We can then share the resulting URL with our partner agencies. Thoughts?

**Comment by NoahKunin** [comment]
This is a personal repo with no public domain dedication, so I'm not sure in what context this work is being created. But due to the tag and the reference to 18F, I guess 18F.

If that's the case, don't we already have this functionality as a result my approval and procurement of http://ethn.io/?

**Comment by onezerojeremy** [comment]
We absolutely do have this 90% of this functionality in ethnio (and are relying on ethnio). 

This was a quick weekend proof of concept prototype cooked up to see what it would take to do something similar, but more adaptable and efficient to government rules that seem to have a way of cropping up, such as the anti-deficiency act. 

From what I've seen, it shows promise as a cost-efficient complement, and especially so in the future as we use ethnio on bigger and bigger projects (as you probably recall, ethnio charges by the page view of the website it lives on).

**Comment by andrewmaier** [comment]
![new screener](https://cloud.githubusercontent.com/assets/2970/8582832/314c5d54-2597-11e5-950d-a1084d8a351a.png)
